topicctl_parser/notifications/datadog.py

```diff
@@ -0,0 +1,37 @@
+import click
+import time
+import httpx
+import os
+
+from typing import Any, Dict
+
+
+DD_API_BASE = "https://api.datadoghq.com"
+
+DATADOG_API_KEY = os.getenv("DATADOG_API_KEY") or os.getenv("DD_API_KEY")
+assert DATADOG_API_KEY, "Please set either DATADOG_API_KEY or DD_API_KEY env var."
+
+
+def send_event_payload_to_datadog(payload: Dict[str, Any]) -> None:
+    # API docs: https://docs.datadoghq.com/api/latest/events/#post-an-event
+    res = httpx.post(
+        f"{DD_API_BASE}/api/v1/events",
+        headers={
+            "DD-API-KEY": DATADOG_API_KEY,
+        },
+        json=payload,
+    )
+    res.raise_for_status()
+    click.echo("\nReported the action to DataDog events:")
+    click.echo(res.json()['event']['url'])
```

Comment:
Please do not print the output from a method that is supposed to be reusable. 
Return instead and let the CLI script that runs the method to decide how to present the result. 
If you print here you are implicitly saying that this method can only be called by some CLI script that produces the output on stdout and that wants to print `Reported the action to DataDog events:`. You do not need to require this in a reusable library.